segmentation/utils/synthetic_spatial_datagen.py

```diff
@@ -7,33 +7,34 @@
 import xarray as xr
 import copy
 
+from random import seed
+from random import randint
 from segmentation.utils import spatial_analysis_utils as sau
+from segmentation.utils import visualize as viz
 from scipy.spatial.distance import cdist
 from skimage.measure import label
 
 
 def direct_init_dist_matrix(num_A=100, num_B=100, num_C=100,
-                            distr_AB={'mean': 100, 'var': 1}, distr_AC={'mean': 20, 'var': 1},
+                            distr_AB=(10, 1), distr_random=(200, 1),
                             seed=None):
     """
     This function will return a random dist matrix such that the distance between cells
-    of types A and B are overall larger than the distance between cells of types A and C
+    of types A and B are overall shorter than the distance between cells of types A and C
```

Comment:
We can update the docstring here to say that it allows the user to specify one set of distances between cells of pops A and B, and a second set of distances between all other cell pairings.